<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>821</width>
      <height>501</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tabBeds">
     <attribute name="title">
      <string>Beds</string>
     </attribute>
     <widget class="QTableWidget" name="tblBed">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>581</width>
        <height>331</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Number</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabShifts">
     <attribute name="title">
      <string>Shifts</string>
     </attribute>
     <widget class="QTableWidget" name="tblShifts">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>581</width>
        <height>331</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <column>
       <property name="text">
        <string>Staff member</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Start</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>End</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Hours</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>25</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sort:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>301</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabIncidents">
     <attribute name="title">
      <string>Incidents</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>25</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sort:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>301</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblIncidents">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>581</width>
        <height>331</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <column>
       <property name="text">
        <string>Date/time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Monitor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Alert</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Bed</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabStaff">
     <attribute name="title">
      <string>Staff</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>25</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sort:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>301</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblStaff">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>581</width>
        <height>331</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Staff member</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabBays">
     <attribute name="title">
      <string>Bays</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>25</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sort:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>301</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblBays">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>581</width>
        <height>331</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string>Bay</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Bed</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMonitor">
     <attribute name="title">
      <string>Monitors</string>
     </attribute>
     <widget class="QTableWidget" name="tblMonitorTypes">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>721</width>
        <height>331</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Monitor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Unit</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Deflt Max</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Deflt Min</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Danger Max</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Danger Min</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabModules">
     <attribute name="title">
      <string>Modules</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>301</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>25</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sort:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblModules">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>581</width>
        <height>331</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string>Module Number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MOD005622 Hospital Monitoring System</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
